blog/_posts/2016-12-19-team-reactives-@-css-and-js-conf-asia.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Team Reactives @ css and js conf Asia
+layout: post
+created_at: Monday Dec 19 2016
+permalink: blog/2016-12-19-team-reactives-@-css-and-js-conf-asia
+current: blog
+author: Team Reactives
+twitter: @shwetha_ravi3
```

Comment:
as explained in [the contributing guide](https://github.com/rails-girls-summer-of-code/summer-of-code/blob/gh-pages/CONTRIBUTING.md#creating-a-new-blog-post), the twitter handle has to be written **without** the `@` character. can you remove it please?